README.md

```diff
@@ -14,12 +14,12 @@ or run `make` in this directory for a list of helpful commands.
 Maintainers:
 
 * [Gabriel Devenyi][devenyi_gabriel]
-* [Ashwin Srinath][srinath_ashwin]
 * [Colin Morris][colin_morris]
 * [Will Pitchers][will_pitchers]
+* [Gerard Capes][[gerard_capes]
 
 [devenyi_gabriel]: http://software-carpentry.org/team/#devenyi_gabriel
-[srinath_ashwin]: http://software-carpentry.org/team/#srinath_ashwin
 [colin_morris]: https://github.com/colinmorris
 [will_pitchers]: https://software-carpentry.org/team/#pitchers_w
+[gerard_capes]: http://colinmorris.github.io/about
```

Comment:
Incorrect link :( 
The existing ones only take you to the list of instructors, but let's use this for now: https://carpentries.org/instructors/#capes_gerard

Reply:
Oops, there's a broken copy paste on my part :-/